pom.xml

```diff
@@ -205,7 +197,7 @@
 
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-plugin-plugin</artifactId>
+				<artifactId>maven-plugin-report-plugin</artifactId>
```

Comment:
Explanation: It seems previously the reporting was part of the `maven-plugin-plugin` but was later extracted into the dedicated plugin `maven-plugin-report-plugin`, see https://issues.apache.org/jira/browse/MPLUGIN-467.